models/issue_reaction_test.go

```diff
@@ -50,9 +50,10 @@ func TestIssueAddDuplicateReaction(t *testing.T) {
 		Type:  "heart",
 	})
 	assert.Error(t, err)
-	assert.Nil(t, reaction)
+	assert.Equal(t, ErrReactionAlreadyExist{Reaction: "heart"}, err)
```

Comment:
Why did you choose `heart` for this test?

Reply:
@bagasme because it is already used [Link](https://github.com/go-gitea/gitea/blob/master/models/issue_reaction_test.go#L50)